main/acle.md

```diff
@@ -1123,6 +1127,8 @@ available. This feature also implies support for the Q flag.
 ACLE 2.0 for A-profile. They are fully supported for M and R-profiles.
 This macro is defined for AArch32 only.
 
+<span id="32-bit-simd-instructions"></span>
```

Comment:
This might be a bit of a nuisance, but in other places where we have added this `span` mechanism I have added a comment to make sure it doesn't get accidentally removed. May I ask you to do the same?